cs/src/Contracts/TunnelEndpoint.cs

```diff
@@ -61,6 +61,13 @@ public abstract class TunnelEndpoint
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public string? PortUriFormat { get; set; }
 
+    /// <summary>
+    /// Gets or sets the default URIs without the port information in the url
+    /// and the web forwarding forwards requests to this url to default port.
+    /// </summary>
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public string? DefaultWebUri { get; set; }
```

Comment:
I want to be careful about consistent naming here because this is a public API. This is a little inconsistent now because it has "web" while `PortUriFormat` does not. Also I'd prefer "tunnel" instead of "default" since I think it's unclear what "default" is referring to. So how about these 4 property names?

```
TunnelUri
PortUriFormat
TunnelSshCommand
PortSshCommandFormat
```

I also considered `TunnelWebUri` / `PortWebUriFormat`, but it's not worth a breaking change to `PortUriFormat`.